.clang-format

```diff
@@ -2,10 +2,10 @@
 Language: Cpp
 BasedOnStyle: Google
 
-ColumnLimit: 120
+ColumnLimit: 80
 NamespaceIndentation: Inner
 
 # Scaled by a factor of 2 such that the base indent is 4
-AccessModifierOffset: -2
+AccessModifierOffset: -3
```

Comment:
Is Google's style "indent by one", or "indent by half indent width"?

I have always interpreted it as the latter, but have never checked.

Reply:
https://google.github.io/styleguide/cppguide.html#Class_Format says "indent by 1", though of course they also say "indent by 2". FWIW, https://github.com/google/styleguide complains about private/public/etc being 2, but not about the overall indent being 4.